sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormModel.cs

```diff
@@ -112,25 +123,22 @@ private static IReadOnlyList<CustomFormSubmodel> ConvertFromLabeled(Model model)
         private static IReadOnlyList<TrainingDocumentInfo> ConvertToTrainingDocuments(TrainResult trainResult)
         {
             var trainingDocs = new List<TrainingDocumentInfo>();
-            if (trainResult?.TrainingDocuments != null)
+            foreach (var docs in trainResult.TrainingDocuments)
```

Comment:
Following the same pattern of other `ConvertTo...` methods across the client: let the constructor do the null check and assume argument to be non-null.